HQSmokeTests/testPages/home/home_page.py

```diff
@@ -41,7 +43,15 @@ def dashboard_menu(self):
         assert self.DASHBOARD_TITLE == self.driver.title, "This is not the Dashboard page."
 
     def reports_menu(self):
-        self.wait_to_click(self.reports_menu_id)
+        try:
+            self.wait_to_click(self.reports_menu_id)
+        except TimeoutException:
+            if self.is_displayed(self.show_full_menu_id):
+                self.click(self.show_full_menu_id)
```

Comment:
Why use simply `click` here and not `wait_to_click`?

Reply:
I guess I would like to take a step back and ask when you'd use `click` vs `wait_to_click`?

I presume `wait_to_click` would typically depend on some element that first needs to be loaded, while `click` could operate on elements that you know would/should exist on the page?

Reply:
`wait_to_click` waits for the interacting element to be clickable. It will wait until the timeout defined, otherwise it will fail. On the other hand, `click()` action will happen as soon as we reach that line of code.